<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>591</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>541</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>311</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Адрес:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
